### DISCUSSION TITLE
WoR-flasher

### DISCUSSION THREAD

**Discussion by dalpets** [community/community#57689] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have completed the The WoR flasher process for Win 10 on Raspberry Pi 400 & everything seems OK until before reboot when the final windows 'setup/Log' screen is overlaid with a message that the process could not find the installation disk, although during the setup I indicated that it is a Kingston 240 gig SSD. That error comes as a surprise!
There was also an earlier message in the process that the partition table was not properly aligned for best performance.
The overlaid message has frozen the final 'Windows on Raspberry' window & there is no mouse or other means to send a log for diagnosis.

Could I please have some help on this in the absence of a log file, 

Thanks

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/Botspot/wor-flasher? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
